src/pages/Error404Page.vue

```diff
@@ -1,41 +1,208 @@
+                console.log(this.$q.screen.lt.md);
+            },
+        },
+    },
+});
 </script>
 
 <template>
 <div
-    class="fullscreen bg-blue text-white text-center q-pa-md flex flex-center"
+    :class="{
+        'c-404-page': true,
+        'landscape' : isMobile && isLandscape
+    }"
 >
-    <div>
-        <div class="main-text">
-            404
+    <div class="c-404-page__video-wrapper">
+        <video
+            id="bgvid"
+            playsinline
+            autoplay
+            muted
+            loop
+            poster="~/assets/background/Video-top_compressed-poster-00001.jpg"
+            class="c-404-page__video"
+        >
+            <source
+                src="~/assets/background/Video-top_compressed-transcode.webm"
+                type="video/webm"
+            >
+            <source
+                src="~/assets/background/Video-top_compressed-transcode.mp4"
+                type="video/mp4"
+            >
+        </video>
+    </div>
+    <div class="c-404-page__video-overlay" ></div>
+    <div class="c-404-page__video-overlay 404-page__video-overlay--shaded" ></div>
+
+    <div class="c-404-page__layout q-pa-lg q-gutter-y-md q-mx-auto">
+        <div class="c-404-page__layout-r1">
+            <div class="c-404-page__layout-logo">
+                <img
+                    class="c-404-page__logo"
+                    src="~assets/logo--telos-wallet.svg"
+                    :alt="$t('home.wallet_logo_alt')"
+                    :aria-label="$t('nav.go_home')"
+                >
+            </div>
+            <div class="c-404-page__layout-title">
+                <div class="c-404-page__title">
+                    404
+                </div>
+            </div>
         </div>
 
-        <div class="text-h2 oops">
-            {{$t('error404.opps')}}
+        <div class="c-404-page__layout-r2">
+            <div class="c-404-page__layout-r2-c1">
+                <h2 class="c-404-page__subtitle">
+                    {{ $t('error404.subtitle') }}
+                </h2>
+            </div>
         </div>
 
-        <q-btn
-            class="q-mt-xl"
-            color="white"
-            text-color="blue"
-            unelevated
-            to="/"
-            label="Go Home"
-            no-caps
-        />
+        <div class="c-404-page__layout-r3">
+            <div class="c-404-page__layout-r3-c1">
+                <p class="c-404-page__text">
+                    {{ $t('error404.text') }}
+                </p>
+            </div>
+        </div>
+
+        <div class="c-404-page__layout-r4">
+            <div class="c-404-page__layout-r4-c1">
+                <q-btn color="primary" to="/" class="c-404-page__button">
+                    {{ $t('error404.take_me_to_home') }}
+                </q-btn>
+            </div>
+        </div>
     </div>
+
 </div>
 </template>
 
 <style lang="scss" scoped>
-.main-text {
-    font-size: 30vh;
-}
 
-.oops {
-    opacity: 0.4;
+.c-404-page {
+
```

Comment:
```suggestion
    max-height: 100vh;
    overflow: hidden;
```

because currently you can scroll the page to a blank white area

Reply:
good catch! didn't notice that. Thanks!